frontend/src/components/Feature/Feature.css

```diff
@@ -10,11 +10,11 @@
 .featureGuides .container-1040 {
   display: flex;
   flex-direction: column;
-  row-gap: 40px;
+  gap: 40px;
```

Comment:
@sence1roandayne the usage of row-gap is preferred on certain japanese ios browsers instead of using gap, Sarah-san taught us on our previous training.

Reply:
@christian-barrios995 Ooohh I see. I will create another ticket to update all gaps to column-gap and row-gap. Thank you